lib/api.js

```diff
@@ -73,7 +73,7 @@ function deleteHooks(reponame, url, token, callback) {
     .get(apiUrl)
     .set('Authorization', 'token ' + token)
     .set('User-Agent', 'StriderCD (http://stridercd.com)')
-    .end(function (res) {
+    .end(function (res) { //CHECK: have a callback with the first param as err and process that err
```

Comment:
Oh ... sorry. The comments are TODOs for me since this pull request is a WIP.
Superagent .end expects a callback with two arguments, err and res, if it is sent a callback function with length that is not equal to two, it just sends us the res object as the only arg.
I am unsure whether we should stick to the (err, res) convention so that we can get the error object from superagent if any. What do you think?